internal/svc/admin_svc.go

```diff
@@ -102,6 +104,81 @@ func (cas *CerbosAdminService) AddOrUpdateSchema(ctx context.Context, req *reque
 	return &responsev1.AddOrUpdateSchemaResponse{}, nil
 }
 
+func (cas *CerbosAdminService) ListSchemas(ctx context.Context, req *requestv1.ListSchemasRequest) (*responsev1.ListSchemasResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	schemaIds, err := cas.store.ListSchemaIDs(ctx)
+	if err != nil {
+		return nil, status.Error(codes.NotFound, fmt.Sprintf("failed to list schema ids: %s", err.Error()))
+	}
+
+	sortSchemas(schemaIds)
+
+	return &responsev1.ListSchemasResponse{
+		SchemaIds: schemaIds,
+	}, nil
+}
+
+func (cas *CerbosAdminService) GetSchema(ctx context.Context, req *requestv1.GetSchemaRequest) (*responsev1.GetSchemaResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	schemas := make([]*schemav1.Schema, 0, len(req.Id))
+	for _, id := range req.Id {
+		sch, err := cas.store.LoadSchema(context.Background(), id)
+		if err != nil {
+			return nil, status.Error(codes.Internal, fmt.Sprintf("could not get schema with id %s: %s", id, err.Error()))
```

Comment:
In these cases, we should log the error to help us debug it. But, the response back to the user should not include the actual error because that can sometimes reveal sensitive information about the server. See the suggestion I made above in another comment.